frontend/src/routes/console.tsx

```diff
@@ -56,12 +54,8 @@ export const ConsoleRoutes = () => {
         path={ROUTES.Console.CUSTOM_PAGES}
         component={CustomPages}
       />
-      {/* Below routes are for dummy implementation for Logo update component and they will be removed after implementation */}
-      {/* <PrivateRoute
-        exact
-        path={ROUTES.Console.NEW_LOGOS}
-        component={newLogos}
-      />
+      <PrivateRoute exact path={ROUTES.Console.NEW_HOME} component={newLogos} />
+      {/*
```

Comment:
should we just make this 
```suggestion
      <PrivateRoute exact path={ROUTES.Console.HOME} component={newLogos} />
      {/*
```
 and remove the redirection?

What do you think?

This we might have to do anyhow if we merge Footer and other components in?

Reply:
Yeah sure. Changed :-)